### PR TITLE
detect if user is an admin

### PR DESCRIPTION
**Pull request by luszczka**

### CODE REVIEW

File: src/Routes/Root/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -21,6 +21,8 @@ const LoginPage = (): ReactElement => {
     setPassword(inputValue.target.value);
   };
 
+  console.log(data);
```
Comment (misiq) [inline src/Routes/Root/LoginPage/LoginPage.tsx]: Clog

File: src/utils/isAdmin.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import useSessionProvider from '../hooks/useSessionProvider';
+
+export const isAdmin = (): boolean => {
+  const { session } = useSessionProvider();
```
Comment (misiq) [inline src/utils/isAdmin.ts:4]: You use here hook so it's not utility but another hook. Also, you can make hone hook that returns user info (nickname avatar, etc) and includes info about the role

File: src/Routes/Root/Protected/Dashboard/Dashboard.tsx [hunk 3]
```diff
@@ -1,13 +1,16 @@
 /* eslint-disable @typescript-eslint/no-misused-promises */
 import { type ReactElement } from 'react';
 import useSignOut from '../../../../hooks/useSignOut';
+import { isAdmin } from '../../../../utils/isAdmin';
```
Comment (misiq) [inline src/Routes/Root/Protected/Dashboard/Dashboard.tsx:4]: Make it absolute

**Review by misiq — CHANGES_REQUESTED** [review]
Good job, small suggestions only

File: src/utils/isAdmin.ts (hunk 2 above)
Comment (luszczka) [reply src/utils/isAdmin.ts:4]: I want to dig deeper on this outside of this mr

**Pull request merged**